src/test/kotlin/org/virtuslab/pulumikotlin/codegen/step2intermediate/PulumiNameTest.kt

```diff
@@ -27,25 +28,28 @@ internal class PulumiNameTest {
         TypeName(
             "aws:acm/CertificateOptions:CertificateOptions",
             namingConfigurationWithSlashInModuleFormat("aws"),
-            PulumiName("aws", listOf("com", "pulumi", "aws", "acm"), "CertificateOptions"),
+            PulumiName("aws", null, listOf("com", "pulumi"), "acm", "CertificateOptions", false),
```

Comment:
(minor)

Would it be possible to get rid of this `null` and `false`? 

Another `PulumiName` factory method might be the solution here?